go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/asecurityteam/transportd
 
-go 1.13
+go 1.16
```

Comment:
this is a worthy upgrade but consumers of transportd should be aware they'll need to bump their go versions as well. We can possibly signify this via a major version upgrade of the version number(git tag).

Reply:
I can do a major version upgrade. If it makes you feel any better sdcli is already on v1.16 of golang so that shouldn't be an issue (which is why the integrations tests run fine here). This is a minor golang version upgrade so I don't think any other upgrades should cause any issues.

Reply:
Yeah in this case I think a major version upgrade is warranted. We do still have some older apps, like kvstore-cidr-aws, that are go 1.12 and import transportd in go.mod. 

Ideally sdcli would match an apps version but its technically used via a separate container and not imported into the code directly so might not be as big a deal, some of these older apps might still be pointing to an older sdcli version.

Reply:
Because Go is backwards compatible and upgrading is considered safe in virtually all cases, most projects don't do a major rev when updating the version of Go they use. For example, the older service should be able to upgrade to 1.16 at the same time as upgrading the library without any negative effects. On the other hand, the Go module system sort of discourages against non-v1 modules. For example, going v2 without embedding all the content in a v2 subdirectory breaks the typical relationship between the import path and the source code layout. It also requires changing both the module path to include a `/v2` suffix in addition to using a `v2.x.x` tag naming scheme. None of it is really a big deal as much as bit of a chore.

That being said, I'm not particularly picky about it and can easily adjust my `go.mod` to bring in a v2.

Reply:
If migrating to 1.16 is not a major change then I am totally fine with a minor version bump here.